python-threatexchange/threatexchange/signal_type/signal_base.py

```diff
@@ -95,7 +95,21 @@ def get_examples() -> t.List[str]:
         return []
 
 
-class TextHasher:
+class FileHasher:
+    """
+    This class can hash files.
+
+    If also inheriting from StrHasher, put this second in the inheritance
+    to prefer file hashing to reading the file in as a Str.
```

Comment:
(Ignorable): I think this is referring to an old implementation - I think it's safe to remove.